Components/App.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { ChangeEvent, useState, FC } from 'react';
+import { ITask } from './interfaces/ITask';
+import TodoTask from './ToDoTask/ToDoTask';
+import CustomDiv from './CustomDiv/CustomDiv';
+import CustomButton from './CustomButton/CustomButton';
+import CustomInput from './CustomInput/CustomInput';
+
+const App: FC = () => {
+  const [task, setTask] = useState<string>("");
+  const [deadline, setDeadline] = useState<number>(0);
+  const [todoList, setTodoList] = useState<ITask[]>([]);
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>): void => {
+    if (event.target.name === "task") {
+      setTask(event.target.value);
+    } else {
+      setDeadline(Number(event.target.value));
+    }
+  };
+
+  const addTask = (): void => {
+    const newTask = { taskName: task, deadline: deadline };
+    setTodoList([...todoList, newTask]);
+    setTask("");
+    setDeadline(0);
+  };
+
+  const completeTask = (taskNameToDelete: string): void => {
+    setTodoList(todoList.filter((task) => {
+      return task.taskName !== taskNameToDelete;
+    }));
+  };
+
+  return (
+    <CustomDiv>
+      <CustomDiv style={{ 
```

Comment:
as we have seen we have always common styles that should be given to custom div like flex direction and alignItems and justify contents .. so lets set these styles inside Custom div and pass there values as props

Reply:
Done